src/utils/router_util.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Route, Redirect, withRouter } from 'react-router-dom';
+
+const msp = state => {
```

Comment:
Also, can we do something like `mapStateToProps` and `mapDispatchToProps` instead? Always good to be descriptive on names.

Reply:
made descriptive names